tasks/fh-build.js

```diff
@@ -70,13 +69,15 @@ module.exports = function(grunt) {
             });
     }
 
-    var fhignore = grunt.template.process('<%= fhignore %>');
+    var fhignore = grunt.config.get('fhignore');
```

Comment:
In `grunt@1.0.0` if processing a template failed to resolve something (in this case, `fhignore` may not be defined in a `Gruntfile.js` of a project), then the command fails. Our implementation here depends on it ignoring that (we sanity check type below on line 74), and `grunt.config.get` seems fine with returning `undefined` as we expect if nothing is there.